tests/test_combine_slices.py

```diff
@@ -30,6 +30,12 @@ def test_single_slice_spacing(self, axial_slices):
         assert np.array_equal(array, dataset.pixel_array.T[:, :, None])
         assert np.isclose(np.linalg.norm(affine[:, 2]), np.abs(slice_spacing))
 
+    def test_rgb_axial_set(self, axial_rgb_slices):
+        combined, _ = combine_slices(axial_rgb_slices)
+
+        manually_combined = np.stack([ds.pixel_array for ds in axial_rgb_slices], axis=0).T
```

Comment:
Is the transpose expected? Perhaps it was necessary because of the `na` and `nb` swap?

Reply:
It is expected because we return the transposed pixel array:
https://github.com/innolitics/dicom-numpy/blob/f0cc77d2b9932cd87326d848833e0448cc7956eb/dicom_numpy/combine_slices.py#L98

Whose shape is more convenient:
`(num_rows, num_rows, num_slices)`
Or:
`(num_channels, num_rows, num_rows, num_slices)`